tests/integration/linux_bridge_test.py

```diff
@@ -31,8 +31,11 @@
 from .testlib.iproutelib import ip_monitor_assert_stable_link_up
 from .testlib.statelib import show_only
 from .testlib.statelib import INTERFACES
+from .testlib.statelib import get_all_macs
 
 TEST_BRIDGE0 = 'linux-br0'
+TEST_BRIDGE0_PORT0 = 'eth1'
```

Comment:
There are a lot of changes that seem to be unintended and revert other commits, please check this.